topology/California Institute of Technology/Caltech CMS Tier2/CIT_CMS_T2_downtime.yaml

```diff
@@ -980,3 +980,16 @@
   Services:
   - CE
 
+# ---------------------------------------------------------
+
+- Class: SCHEDULED
+  ID: 499559472
+  Description: Move CEs to container
```

Comment:
Cool! Are you guys using our Hosted CE container or something else?

Reply:
Not sure if Hosted CE would work for us. We have many custom things and different clusters/networks with flocking enabled on multiple schedds. Also many other local users and allow to submit to GPU resources/CPU Resources competing together with CMS/OSG fair share. Same as custom things to monitor CEs and what is allowed to submit :)
We go two ways so far: docker and OpenVZ.

Reply:
Interesting! Are the docker containers completely homegrown or are you basing them off of official OSG or HTCondor containers?